clayui.com/gatsby/onCreateNode.js

```diff
@@ -78,7 +78,7 @@ module.exports = exports.onCreateNode = ({actions, getNode, node}) => {
 						'@clayui/',
 						''
 					)}.html`;
-				} else {
+				} else if (mainTabURL) {
```

Comment:
@kresimir-coko I see that all the main tabbed pages are using the wrong information at the top of the document, they are being pointed to `Button Group`. Maybe here we should ignore it when the page has `mainTabURL`?

```suggestion
				} else if (!mainTabURL) {
```

Reply:
Interesting @matuzalemsteles, I can't replicate that locally, on dev or prod server. I'll try the suggestion

Reply:
Hmm, this is really weird.